File: Leetcode/1295/Linear_Search_Algo/main.cs
public class Solution {
    public int FindNumbers(int[] nums) {
        int count = 0;
        foreach (int num in nums) {
            int numm = (int) Math.Log10(num) + 1;
            if (numm%2==0) count++;
        }
        
        return count;
    }
}

File: Leetcode/1295/Linear_Search_Algo/main.swift
class Solution {
    func findNumbers(_ nums: [Int]) -> Int {
        var count: Int = 0
        for num in nums {
            let numm = Int(log10(Double(abs(num))) + 1)
            if (numm%2==0) {
                count+=1
            }
        }       
        return count
    }
}